stable_baselines/common/vec_env/base_vec_env.py

```diff
@@ -239,8 +240,11 @@ def getattr_recursive(self, name):
         :param name (str) name of attribute to look for
         :return: (object) attribute
         """
-        if name in self.__dict__:  # attribute is present in this wrapper
-            attr = self.__dict__[name]
+        class_attributes = {k: v for k, v in inspect.getmembers(self.__class__)}
+        all_attributes = self.__dict__.copy()
+        all_attributes.update(class_attributes)
+        if name in all_attributes:  # attribute is present in this wrapper
```

Comment:
Rather than trying to predict whether an attribute is present here or in a superclass, is there a way of checking directly? We can't call `getattr` since that will invoke `__getattr__` and trigger an infinte recursion, but would `__getattribute__` do the job?